fontforge/scripting.c

```diff
@@ -3278,9 +3281,8 @@ static void bLoadTableFromFile(Context *c) {
 	sf->ttf_tab_saved = tab;
     } else
 	free(tab->data);
-    tab->len = len;
```

Comment:
It's used to `malloc` the space required and for the `fread` length. Since `fread` may read less bytes than what is specified, `tab->len` is set to the result of `fread`